Example/PINRemoteImage Tests/PINRemoteImage_Tests.m

```diff
@@ -10,7 +10,9 @@
 #import <XCTest/XCTest.h>
 #import <PINRemoteImage/PINRemoteImage.h>
 #import <PINRemoteImage/UIImageView+PINRemoteImage.h>
+#if __has_include(<FLAnimatedImage/FLAnimatedImage.h>)
```

Comment:
Should we use `USE_FLANIMATED_IMAGE` to control these includes, too? It seems like that would be more consistent and easier to control from the top-level configuration.